### ISSUE TITLE
OACMPDM6 bevel shadows

### ISSUE THREAD

**Issue by The-Gig** [NeonKnightOA/oacmpvol1#13]
All around the map there are some areas where the edges of the bevels seem to show some strange shadows. Not really a major problem, just wondering if maybe recompiling with different lighting options may fix?
Also, in the third screenshot, I wonder if Jan really meant to use that texture there of it is an error and he wanted to use the same as the rest of the wall... I don't know, as around this map there are some thin vertical textures different from the surrounding ones surely on purpose.
![shot0222](https://user-images.githubusercontent.com/23258514/161794496-9a53ac59-e6d6-4b22-8383-e4802e7546e6.jpg)
![shot0223](https://user-images.githubusercontent.com/23258514/161794500-b021301e-1571-4659-9d9c-92da5713f816.jpg)
![shot0224](https://user-images.githubusercontent.com/23258514/161794503-5a8a57d8-f23e-43c9-93b2-59a8e5ada9f3.jpg)

**Comment by NeonKnightOA** [comment]
Yeah, those bevels do look weird. I'll see what can I do about them.

**Comment by NeonKnightOA** [comment]
Jan's maps will take a bit longer to fix, they use all kinds of irregular brushes. There's a reason why every mapping guide for Radiant recommends sticking with high integer numbers for mapping. It helps reducing a lot of errors.

**Comment by NeonKnightOA** [comment]
For the next version I'm going to change those textures into trims.

**Comment by The-Gig** [comment]
Looking at those same places with version 20230430:
![shot0044](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/1067ead7-dac2-4038-9915-d8f808a733d6)
![shot0039](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/fde5fb98-cb27-46cd-9edf-9c0c60eca9c9)
![shot0041](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/649f9d3f-6c8d-416d-802f-becb5102bb8d)
Those odd shadows seem to me less noticeable than before. I hope it's not just because I'm using a different PC now. Maybe they could just be left this way?

However, looking at them, I noticed a couple of possible _texturing issues_:
This is located near the first screenshot above. Just at the left of the crosshair, you can see that sort of "stripe" with a gray lighter than the surrounding brushes or patches. Not a big problem, I guess.
![shot0042](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/42a96462-fa93-4a6a-9f2c-7973b292f3d7)

This one is located near the place of the second screenshot above. On the wall on its upper floor. You see, there is a bezier curve which seems to have it texture _badly stretched_, which looks quite bad. Maybe a bit off-topic for this "issue" as in this case it isn't a problem of shadow or lighting, I'm sorry!
![shot0043](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/a4bdeed3-2bf9-41aa-9167-3d477b3ba93b)
